### PR TITLE
SDA

### PR DESCRIPTION
**Pull request by MelissaAutumn**
I've squashed this branch ahead of time :smile: 

Please let me know if you think I'm missing something important.

### CODE REVIEW

File: docs/topics/api/reviewers.rst [hunk 1]
```diff
@@ -76,9 +75,10 @@ add-on.
...
        permission.
 
-.. http:patch::/api/v4/reviewers/addon/(int:addon_id)/flags/
+.. http:patch:: /api/v4/reviewers/addon/(int:addon_id)/flags/
 
     :>json boolean auto_approval_disabled: Boolean indicating whether auto approval are disabled on an add-on or not. When it's ``true``, new versions for this add-on will make it appear in the regular reviewer queues instead of being auto-approved.
     :>json string|null pending_info_request: Deadline date for the pending info request as a string, or ``null``.
     :>json boolean needs_admin_code_review: Boolean indicating whether the add-on needs its code to be reviewed by an admin or not.
     :>json boolean needs_admin_content_review: Boolean indicating whether the add-on needs its content to be reviewed by an admin or not.
+    :>json boolean needs_sensitive_access_review: Boolean indicating whether the add-on needs its content to be reviewed by an admin or not. This is related to the `Addon.require_sensitive_data_access` property.
```
Comment (jobisoft) [inline docs/topics/api/reviewers.rst:84]: These reviews should show up in the regular review queue. All we want is to disable auto approval for add-ons which have access to sensitive data.

I do not think we need to escalate them to admin review directly.

I assume this is just a wording issue here, and not a code logic issue, right?

File: src/olympia/addons/tests/test_tasks.py [hunk 2]
```diff
@@ -315,3 +315,82 @@ def test_basic(self):
...
+        AppVersion.objects.get_or_create(
+            application=amo.THUNDERBIRD.id, version='60.0')  # Default Min Version
+        AppVersion.objects.get_or_create(
+            application=amo.THUNDERBIRD.id, version='102.0') # Targeted Max Version
+        AppVersion.objects.get_or_create(
+            application=amo.THUNDERBIRD.id, version='*')
+
+
+    def test_addon_with_sda(self):
+        """Test addon with sensitive permissions. This should cause both the Admin and AdminReviewer Flags to be True"""
```
Comment (jobisoft) [inline src/olympia/addons/tests/test_tasks.py:341]: The test is checking for `requires_sensitive_data_access` and `needs_sensitive_data_access_review`, but not any admin flags. Is this just a wording issue here, or a real code logic issue? Sensitive data does not need to escalate to admin review, just disable auto approval.

File: src/olympia/addons/tests/test_tasks.py [hunk 3]
```diff
@@ -315,3 +315,82 @@ def test_basic(self):
...
+        assert addon_without_sda.requires_sensitive_data_access == False
+        assert not addon_without_sda.needs_sensitive_data_access_review
+
+        migrate_addons_that_require_sensitive_data_access(addon_without_sda)
+
+        assert addon_without_sda.requires_sensitive_data_access == False
+        assert not addon_without_sda.needs_sensitive_data_access_review
+
+    def test_addon_with_sda_and_exfiltrate(self):
+        """Test addon with sensitive permissions, but also an explicitly set `exfiltrate` permission. The Addon Flag should be True, but the AdminReviewer Flag should not"""
```
Comment (jobisoft) [inline src/olympia/addons/tests/test_tasks.py:379]: This again refers to an AdminReviewer flag, but I do not see it used in the test.

**Review by jobisoft — COMMENTED** [review]
I think this looks good. The comments I made should all just be about a wording issue but not a real code logic issue: AddonReviewer Flag vs. AdminReviewer Flag. Right?

**Comment by MelissaAutumn** [comment]
Corrected the misleading comments, simplified some loops, and fixed a few issues a new test run encountered. Looking good for staging.

**Pull request merged**